sdk.yaml

```
@@ -229,6 +229,12 @@ Realtime:
       .synopsis: |
         Get the `id` of the connection, which is a public string provided by the Ably service and is unique for this connection.
         Unavailable until connected.
+    Incremental Backoff:
+      .specification: RTB1
+      .synopsis: |
+        Use backoff and jitter coefficients to make the constant `disconnectedRetryTimeout` and `channelRetryTimeout` values variable at runtime.
```

I think the wording here is a bit off - making 'constant' values 'variable' doesn't sound right to me.

```suggestion
        Use backoff and jitter coefficients to vary the `disconnectedRetryTimeout` and `channelRetryTimeout` values at runtime.
```